atac_snvs/1.0/atac_snvs.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+
+
+
+##### ATTRIBUTION #####
+
+# MBB 498 Project - Rachel LaFrance
+
+
+
+##### ABOUT #####
```

Comment:
Thanks for the description!
Can we also add here an example of the running command for this script?